components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.common/src/main/java/org/wso2/carbon/identity/api/server/userstore/common/UserStoreConstants.java

```diff
@@ -105,7 +106,13 @@
         ERROR_CODE_REQUEST_BODY_NOT_FOUND("60010", "Invalid userstore update request",
                 "Userstore update request is either NULL or empty"),
         ERROR_CODE_USER_STORE_LIMIT_REACHED("60011", "Unable to create a user store.",
-                "Maximum number of allowed user stores have been reached.");
+                "Maximum number of allowed user stores have been reached."),
+        ERROR_CODE_ERROR_UPDATING_CLAIM_MAPPING("60012", "Unable to add local claims",
+                "Error occurred while adding local claims"),
+        ERROR_CODE_ERROR_RETRIEVING_CLAIM_MAPPING("60013", "Invalid attribute mapping.",
+                "Invalid userstore provided for attribute mapping"),
```

Comment:
This is thrown when an error occurred while getting claims. Does that mean its an invalid userstore?

Reply:
Addressed with commit c731518fd0ea44d2d06c025d044e61114470